`_example/main.go` raises panic

I cloned this repo and built example package under `/_example`. When `_example` is executed, it raises panic below.

```bash
~/go-mysql-server/_example$ go build
~/go-mysql-server/_example$ ./_example
panic: Actual Value Type: []string, Expected Value Type: sql.JSONValue

goroutine 1 [running]:
github.com/dolthub/go-mysql-server/memory.(*Table).verifyRowTypes(0xc000761e60, {0xc000370980, 0x0?, 0xc000370980?})
        /home/syuparn/go-mysql-server/memory/table.go:1691 +0x236
github.com/dolthub/go-mysql-server/memory.(*tableEditor).Insert(0xc00077f9f0, 0xdb2c20?, {0xc000370980?, 0x4, 0x4})
        /home/syuparn/go-mysql-server/memory/table_editor.go:124 +0xa5
github.com/dolthub/go-mysql-server/memory.(*Table).Insert(0xedd700?, 0x1?, {0xc000370980, 0x4, 0x4})
        /home/syuparn/go-mysql-server/memory/table.go:579 +0x4c
main.createTestDatabase()
        /home/syuparn/go-mysql-server/_example/main.go:77 +0x846
main.main()
        /home/syuparn/go-mysql-server/_example/main.go:43 +0x2e
```

This seems due to a type mismatch of `sql.NewRow` arguments.

https://github.com/dolthub/go-mysql-server/blob/main/_example/main.go#L77

```go
table.Insert(ctx, sql.NewRow("John Doe", "john@doe.com", []string{"555-555-555"}, creationTime))
```

--- a/.gitignore
+++ b/.gitignore
@@ -3,6 +3,9 @@
 *.a
 *.so
 
+# Compiled binaries
+_example/_example
+
 # Folders
 _obj
 _test

--- a/_example/main.go
+++ b/_example/main.go
@@ -43,6 +43,7 @@ func main() {
 			createTestDatabase(),
 			information_schema.NewInformationSchemaDatabase(),
 		))
+	engine.Analyzer.Catalog.MySQLDb.AddRootAccount()
 
 	config := server.Config{
 		Protocol: "tcp",
@@ -74,9 +75,9 @@ func createTestDatabase() *memory.Database {
 	creationTime := time.Unix(1524044473, 0).UTC()
 	db.AddTable(tableName, table)
 	ctx := sql.NewEmptyContext()
-	table.Insert(ctx, sql.NewRow("John Doe", "john@doe.com", []string{"555-555-555"}, creationTime))
-	table.Insert(ctx, sql.NewRow("John Doe", "johnalt@doe.com", []string{}, creationTime))
-	table.Insert(ctx, sql.NewRow("Jane Doe", "jane@doe.com", []string{}, creationTime))
-	table.Insert(ctx, sql.NewRow("Evil Bob", "evilbob@gmail.com", []string{"555-666-555", "666-666-666"}, creationTime))
+	table.Insert(ctx, sql.NewRow("John Doe", "john@doe.com", sql.JSONDocument{Val: []string{"555-555-555"}}, creationTime))
+	table.Insert(ctx, sql.NewRow("John Doe", "johnalt@doe.com", sql.JSONDocument{Val: []string{}}, creationTime))
+	table.Insert(ctx, sql.NewRow("Jane Doe", "jane@doe.com", sql.JSONDocument{Val: []string{}}, creationTime))
+	table.Insert(ctx, sql.NewRow("Evil Bob", "evilbob@gmail.com", sql.JSONDocument{Val: []string{"555-666-555", "666-666-666"}}, creationTime))
 	return db
 }

